app/views/dashboard/index.html.erb

```diff
@@ -1,4 +1,6 @@
 <h2><div class="page-title">Merchant Dashboard</div></h2>
 
 
-<h3><%= @merchant.name %></h3>
\ No newline at end of file
+<h3><%= @merchant.name %></h3>
+<p><%= link_to "My Items", "/merchants/:merchant_id/items" %><p>
+<p><%= link_to "My Invoices", "/merchants/:merchant_id/invoices" %></p>
```

Comment:
Just looked over US 2 and I think you might have missed the link to the dashboard: 
`When I visit my merchant dashboard (/merchants/:merchant_id/dashboard)`

Also, don't forget that you can now leverage URL pathways instead and do something like:
`merchant_dashboard_index_path`
`merchant_items_path`
`merchant_invoices_path`

Reply:
I'm looking over the US again
```
As a merchant,
When I visit my merchant dashboard (/merchants/:merchant_id/dashboard)
Then I see link to my merchant items index (/merchants/:merchant_id/items)
And I see a link to my merchant invoices index (/merchants/:merchant_id/invoices)
```
and I don't see anything about a link to the dashboard, only that I see two links FROM the dashboard.

Reply:
Omg, @dani-wilson  I'm so sorry. My eyes played tricks on me - I think I haven't been in the best headspace today. 🤯 

This is perfect! Approving now

Reply:
@westonio no problem! Thanks for looking over that for me, I appreciate you wanting to be thorough! :))